### ISSUE TITLE
Driver isn't installing

### ISSUE THREAD

**Issue by Ferdous123** [shibajee/realtek-uad-nahimic-mod#12]
Hello, I've been using "realtek-uad-nahimic-mod" for almost a year. But recently I updated the windows to version 20H2(build 19042.867), It was a feature update so nothing should change but I found that the Nahimic wasn't working(says driver issue)!

**So I tried to install the latest "realtek-uad-nahimic-mod" driver, But it wasn't installing then. If I run the setup.exe, nothing happens. I even tried to install the previously installed driver(the one which worked before windows update) BUT, no success.** 

(Turning off  Driver signature didn't help so I tried test mode with driver signature off. Same result)

What can I do now?

**Comment by shibajee** [comment]
Sorry for the late reply. Actually this mod won't work in future Windows version as the App itself won't work after v1.5.2.0. Even if u force install the v1.5.2.0, windows store will auto update it to the latest version. There is no way to prevent that unless u remove the Store app itself by some third party tool. I'm sorry but this is the END of this project. However there is an alternative which I'm working on now, [Asus Sonic Studio 3](https://www.asus.com/microsite/mb/ROG-supremefx-gaming-audio/sonic_studio.html) . This is literally the Asus modified version of Nahimic, nothing else. I'll release it soon. @Ferdous123

**Comment by Ferdous123** [comment]
Actually your latest driver wasn't installing. I even tried older versions. Nothing happens if I run the setup.exe. Maybe something's wrong with that or, the latest windows version caused something that is preventing the modded drivers setup.exe to work properly....
I found an old unmodded Nahimic driver(v6.0.1.8619), which setup.exe is working perfectly. So I copied the files from win64(didn't replace anything) & the setup.exe to your latest modded driver(v6.0.9079.1) folder. Now it installed perfectly. Plus,  Nahimic & Realtek audio console is working now. 

Though in control panel, the version of the audio driver is showing the unmodded driver version.

As you said, I disabled MS store.
